holoviews/plotting/bokeh/element.py

```diff
@@ -1794,6 +1794,10 @@ def _init_glyph(self, plot, mapping, properties):
         if self._subcoord_overlaid:
             y_source_range = self.handles['y_range']
             if isinstance(self.subcoordinate_y, bool):
+                if "subcoordinate_y" not in y_source_range.tags[1]:
+                    raise RuntimeError(
+                        'Failed retrieving "subcoordinate_y". Labels mismatched for initial and updated DynamicMap plots.'
+                    )
```

Comment:
```suggestion
                    msg =  'Failed retrieving "subcoordinate_y". Labels mismatched for initial and updated DynamicMap plots.'
                    raise RuntimeError(msg)
```

To avoid displaying the message twice, in both the code and the actual exception.

Reply:
Good catch! Thanks.